media/src/rooms.js

```diff
@@ -34,11 +34,15 @@ const makeSocket = function(roomId, handleMessage=null) {
     return socket;
 };
 
-const setHost = (socket=null) => {
+const setHost = (socket=null, objects) => {
+    if (!objects) {
+        objects = [];
+    }
```

Comment:
Minor point, but you can set a param default in the function signature above: 
```
const setHost = (socket=null, objects=[]) => {
```

Reply:
For some reason it was giving me trouble before. So I decided to make it explicit. Working now though, embarrassingly enough.